.travis.yml

```diff
@@ -7,8 +7,11 @@ node_js:
   - "8"
 cache:
   directories:
-    - node_modules
+    - "$HOME/.npm"
```

Comment:
Can you help me understand this change? I'm not super familiar with the npm cache and reading their docs didn't help much.

Reply:
`npm ci` always deletes `node_modules` so we can just use the npm cache ;-)